spec.bs

```diff
@@ -255,7 +252,9 @@ When {{Worklet/addModule()}} is called for a worklet, it will run [=check if add
     1. If |origin| is an [=opaque origin=], then return false.
     1. Let |globalObject| be the [=current realm=]'s [=global object=].
     1. [=Assert=] that |globalObject| is a {{Window}} or a {{SharedStorageWorkletGlobalScope}}.
-    1. If |globalObject| is a {{Window}}, and the result of executing [=is feature enabled|Is feature enabled in document for origin?=] on "[=PermissionsPolicy/shared-storage=]", |globalObject|'s [=associated Document=], and |origin| is "`Disabled`", return false.
+    1. Let |document| be |globalObject|'s [=associated Document=].
```

Comment:
`associated Document` is only a property for `Window`. It seems we should only check the permissions policy when `|globalObject|` is a `Window` (and just skip checking for `SharedStorageWorkletGlobalScope`).

Reply:
Fixed.